### ISSUE TITLE
Fix deprecation in `erc20-counter`

### ISSUE THREAD

**Issue by Wollac** [risc0/risc0-ethereum#293]

**Comment by mahmudsudo** [comment]
can i take on this issue ?

**Comment by nategraf** [comment]
@mahmudsudo, sure! This is a good first issue

**Comment by nategraf** [comment]
@Wollac, did you address this? I no longer see the deprecated call in the example (although the commit above it says "there are two options", and it's unclear what's that's referring to exactly)
https://github.com/risc0/risc0-ethereum/blob/main/examples/erc20-counter/apps/src/bin/publisher.rs#L143-L146

If this is fixed, go ahead and close this issue

**Comment by Wollac** [comment]
@nategraf good call, this was fixed as a byproduct in https://github.com/risc0/risc0-ethereum/pull/316
